packages/fx-core/resource/package.nls.json

```diff
@@ -172,10 +171,12 @@
   "core.collaboration.AzureTip": "\nIf added user cannot access Azure resources, you need to manually setup access policy via Azure portal, refer to this link for more details: ",
   "core.collaboration.ListingM365Permission": "Listing Microsoft 365 permissions\n",
   "core.collaboration.AccountUsedToCheck": "Account used to check: ",
-  "core.collaboration.StartingListAllTeamsAppOwners": "Starting list all teams app owners for environment: ",
+  "core.collaboration.StartingListAllTeamsAppOwners": "\nStarting list all teams app owners for environment: ",
+  "core.collaboration.StartingListAllAadAppOwners": "\nStarting list all Azure Active Directory app owners for environment: ",
   "core.collaboration.M365TeamsAppId": "Microsoft 365 Teams App (ID: ",
-  "core.collaboration.SsoAadAppId": "), SSO AAD App (ID: ",
+  "core.collaboration.SsoAadAppId": "SSO AAD App (ID: ",
```

Comment:
is it a typo for "("?

Reply:
“(” is expected here. We are outputting: 'SSO AAD App (ID: ' + aadAppId + ')'.